tests/test_ivcfmerge_batch.py

```diff
@@ -0,0 +1,25 @@
+from tempfile import NamedTemporaryFile
+
+from hypothesis import given, strategies as st
+
+from ivcfmerge.batch import ivcfmerge_batch
+
+
+@given(batch_size=st.integers(min_value=2))
+def test_merging_example_input(batch_size):
+    input_paths = [
```

Comment:
Can we also add a test case for handling empty input paths list?

Reply:
Please also add a test case for handling list that contains only one input path.

Reply:
What do you want the behavior to be when the input list is empty?

Reply:
by default, `ivcfmerge` (no batch) write an empty file with an empty input list, should we maintain that behavior for batch version?

Reply:
I think when the input list is empty, the function will produce an empty output file. This is probably what the current implementation already does. When the input list contains only one file, the function should produce an output file exactly the same as the input file.